spec/workers/session_stopped_alerts_worker_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe SessionStoppedAlertsWorker do
+  it 'sends email alerts to users whose sessions stopped streaming in the last 30 minutes' do
+    allow(Time).to receive(:current).and_return(Time.parse('2010-01-01'))
+    user =
+      create_user!(session_stopped_alert: true, email: 'useremail@example.com')
+    create_session!(
+      user: user,
+      title: 'Session Title',
+      last_measurement_at: Time.current - 35.minutes
+    )
+    create_session!(last_measurement_at: Time.current - 65.minutes)
+    create_session!(last_measurement_at: Time.current - 25.minutes)
+
+    subject.perform
+
+    expect { subject.perform }.to change {
+      ActionMailer::Base.deliveries.count
+    }.by(1)
+    expect(ActionMailer::Base.deliveries.first.to).to eq(
+      %w[useremail@example.com]
+    )
+    expect(ActionMailer::Base.deliveries.first.subject).to eq(
```

Comment:
Is this the formatting that prettier did or something like that? Because it seems kind of weird in this file

Reply:
Yes, the formatting is fully decided by prettier.